### DISCUSSION TITLE
what happens to the contributions when i leave the org (remove alias email)?

### DISCUSSION THREAD

**Discussion by yblee85** [community/community#38188] [Other Feature Feedback, Questions, & Ideas] [answered]
I have setup alias emails for work, `work-email@emai.com`
and I have been contributing to repo under that email.
gotta say seeing those contributions in profile is quite satisfying :smile: 
but then it occurred to me that what happens to those contributions done by my alias email when i leave the org? (remove alias email)
would that be gone from my profile or would it stay?

Thanks,

**Comment by DSDmark** [comment, marked as answer]
Hi @yblee85,

Once you are kicked out of **organization**, all of these commits will disappear.
Hence, it's important to keep it up-to-date,

Here best practices for leaving your company [**Github Docs**](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/best-practices-for-leaving-your-company)

Also check this out, it could help you [Don't lose your Github contributions when you leave an organization](https://vikrantbhat.hashnode.dev/dont-lose-your-github-contributions-when-you-leave-an-organization)

And there is whole discussion about that:-
(https://github.com/isaacs/github/issues/1138)

**Reply by DSDmark** [reply]
I forget to mention. I'm not sure, but I think you should read this for future reference:-
[**contributions not showing**](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)

**Comment by david-engelmann** [comment]
You could leave the email attached to the account still
